src/Grid/Column/Collector/DataObject/FieldDefinitionCollector.php

```diff
@@ -111,7 +113,9 @@ private function groupDefinitions(
             }
 
             // Use ObjectbricksCollector
```

Comment:
Let's update the comment a bit? Since it now does a bit more? 

```suggestion
            // When definition is instance of ObjectBrick there is a ObjectbricksCollector. We also skip all field types in skipFieldTypes
```